docs/changelog.rst

```diff
@@ -7,26 +7,41 @@ Changelog
 Development version (|development_version|)
 ===========================================
 
-.. |development_version| replace:: 1.2.0rc2
+.. |development_version| replace:: 1.2.0rc3
 
 Dependency changes
 ------------------
 
 * docutils 0.18 is supported. (#1381)
-* Sphinx 6 support added (#1385)
+* Sphinx 6 support added
 * Added ``sphinxcontrib-jquery`` as a dependency (#1385)
 * Python 3.11 is officially supported and tested. (#1395)
 * Python 3.4 and 3.5 are officially not supported (#1395)
 
 
-Note for users of Sphinx<3
+Changes
+-------
+
+* Automatically use ``sphinxcontrib-jquery`` in Sphinx 6+ (#1399)
+* Use new context vars ``logo_url``, ``favicon_url`` and ``root_doc`` when available (#1405)
+
+Note for users of Sphinx<4
 --------------------------
 
 If you cannot use a more recent Sphinx release,
 you should at least ensure you are using the most recent release for your major version.
 Currently, these are Sphinx `1.8.6` and `2.4.5`.
 Older releases may install unsupported versions of several dependencies, including Jinja2 and docutils.
 
+Known issues
+------------
+
+In some cases, jQuery is not loaded with Sphinx 6 on Read the Docs.
+Workaround: You need to add ``sphinx_rtd_theme`` to ``extensions`` in your ``conf.py``.
+See `readthedocs.org issue #9654`_ for updates.
+
+.. _readthedocs.org issue #9654: https://github.com/readthedocs/readthedocs.org/pull/9654
+
```

Comment:
A note on this:

We will release rc3 with this known issue and then work to fix it in RTD before 1.2.0 because we cannot risk users building documentation that _looks_ like it's working but actually doesn't have jQuery and thus have specific small details broken (the RTD Flyout menu for instance)